src/view.rs

```diff
@@ -300,6 +302,11 @@ impl View {
         self.touch();
     }
 
+    /// Clear the view to a color.
```

Comment:
This comment isn't right. You can just remove it as we don't really need a comment here.